log10/feedback/feedback_task.py

```diff
@@ -81,10 +81,18 @@ def get(self, id: str) -> httpx.Response:
 @click.option("--name", prompt="Enter feedback task name", help="Name of the task")
 @click.option("--task_schema", prompt="Enter feedback task schema", help="Task schema")
 @click.option("--instruction", help="Task instruction", default="")
-def create_feedback_task(name, task_schema, instruction):
+@click.option(
+    "--completion_tags_selector",
+    prompt="Enter completion tags selector",
```

Comment:
since this could be None, I think we could set a default value to `""`, and remove the `prompt` part.

Reply:
Thanks @wenzhe-log10 - will make this change